packages/gatsby-source-wikipedia/index.js

```diff
@@ -1 +1,13 @@
 // noop
+
+const { getArticle, search, getMetaData, fetchNodesFromSearch } = require("./src/fetch")
+
+getArticle('Andries Tunru', 'nl').then(res => console.log(res))
+
+search({query: 'kaas', limit: 10, lang: 'nl'}).then(res => console.log(res))
+
+getMetaData('frommage', 'en')
+
+fetchNodesFromSearch({query: 'velo', lang: 'fr'})
+
+console.log('done.')
```

Comment:
What's this? Probably a leftover after some testing? Please remove.

Reply:
This can be removed.